README.rst

```diff
@@ -1,17 +1,15 @@
 smart_importer
 ==============
 
-A smart importer for
-`Beancount <http://furius.ca/beancount/>`__ and
-`Fava <https://github.com/beancount/fava>`__, currently in development
-as a suggestion for `#579 Import: Intelligent suggestions for account
-names <https://github.com/beancount/fava/issues/579>`__
+Decorators for augmenting
```

Comment:
Maybe just "Augment" here - don't think everyone know what 'decorators' are (and the decorating part is not the main thing about this package)

Reply:
yes, that's right, thanks for pointing it out!